editor/modes/visual-editor/block.js

```diff
@@ -73,9 +72,11 @@ function VisualEditorBlock( props ) {
 					} ) ) } />
 			) : null }
 			<BlockEdit
-				isSelected={ isSelected }
+				focus={ focus }
 				attributes={ block.attributes }
-				setAttributes={ setAttributes } />
+				setAttributes={ setAttributes }
+				updateFocus={ onFocus }
```

Comment:
Even just for consistency with `setAttributes`, do you think naming this `setFocus` is still accurate / any better?